Protection events/ExploitGuardNetworkProtectionEvents.txt

```diff
@@ -0,0 +1,6 @@
+//Simple query to show the unique network connections that were audited or blocked by ExploitGuard
+// For more questions on this query, feel free to ping @FlyingBlueMonki on twitter or mattegen@microsoft.com via email
+MiscEvents
+| where  ActionType  =~ "ExploitGuardNetworkProtectionBlocked"
+| summarize count(RemoteUrl) by InitiatingProcessFileName, RemoteUrl,Audit_Only=tostring(parse_json(AdditionalFields).IsAudit)
```

Comment:
Nit picking: please add a space after `RemoteUrl,` and before `AuditOnly`:
`RemoteUrl, AuditOnly`...